docs/execute.adoc

```diff
@@ -166,6 +166,7 @@ Functional units can either be pipelined or iterative.
 Pipelined functional units have fixed execution latencies, so once an operation is issued, it will execute without stalling.
 The sequencing mechanism checks for future write port conflicts on the target VRF bank across inflight and simultaneously sequenced operations to ensure that the next sequenced operation will not induce a structural hazard on the write port in the future.
 If a conflict is detected, the younger operation will be stalled and will likely start executing the very next cycle.
+This allows the operations to then proceed across the banks without any further conflicts with each other, barring other hazards.
```

Comment:
This current phrasing is a bit confusing "without further conflicts, barring other hazards". What are you trying to emphasize here?

Reply:
Rereading it, I think the comment isn't important. It was just meant to emphasize that the operations become offset and will proceed without conflicting with each other again unless some other action in the machine causes them to come into conflict again.